src/utils/__tests__/blockly-api.test.js

```diff
@@ -74,6 +74,26 @@ describe('Blockly API', () => {
     expect(sendToRover).toHaveBeenCalledWith('left-motor:100\n');
   });
 
+  test('handles setMotor with BOTH', () => {
+    const setMotorHandler = interpreter.createNativeFunction.mock.calls[2][0];
+
+    const result = setMotorHandler({ data: 'BOTH' }, { data: 'FORWARD' }, { data: 100 });
+
+    expect(result).toBe(false);
+    expect(sendToRover).toHaveBeenCalledTimes(1);
+    expect(sendToRover).toHaveBeenCalledWith('both-motors:100\n');
+  });
+
+  test('handles setMotor with BACKWARD', () => {
+    const setMotorHandler = interpreter.createNativeFunction.mock.calls[2][0];
+
+    const result = setMotorHandler({ data: 'RIGHT' }, { data: 'BACKWARD' }, { data: -100 });
+
+    expect(result).toBe(false);
+    expect(sendToRover).toHaveBeenCalledTimes(1);
+    expect(sendToRover).toHaveBeenCalledWith('right-motor:100\n');
```

Comment:
Shouldn't this be `-100`?

Reply:
No, it started as `-100`, then the `BACKWARD` made it `+100`.